source/includes/_fiat_to_crypto_flow.md

```diff
@@ -0,0 +1,236 @@
+# Fiat to Crypto flow
+
+This section describes the endpoints used in the fiat to crypto flow. 
+
+## Create Fiat to Crypto Invoice
+
+> Example Call
+
+```shell
+curl -X POST 'https://api.onramp.ltd/rpc/create_fiat_to_crypto_invoice' \
+-H 'Content-Type: application/json' \
+-H 'x-xco-authorization: Bearer 00000000-0000-0000-0000-000000000000' \
+-d '{
+    "fiat_amount": 3000,
+    "fiat_currency": "EUR",
+    "callback_url": "https://www.example.com",
```

Comment:
The difference from fiat ingress flow is that we do not have `payment_ack_url` (no need since after the fiat leg is completed we will initiate the crypto leg) and `user_redirect_url` (this might be needed since maybe we want to redirect the user back to the merchant, WDYT @yowlu), instead there is one `callback_url` for merchants to receive status updates.

Reply:
I think it is ok. In my mind they were the same is just we were calling them at different times, but it is ok if you want to differentiate them.